code/datums/uplink/uplink_items.dm

```diff
@@ -68,10 +69,11 @@ var/datum/uplink/uplink = new()
 	// Making the assumption that if no uplink was supplied, then we don't care about antag roles
 	if(!U || !antag_roles.len)
 		return 1
-
 	if(!U.uplink_owner)
+		to_chat(world, "no uplink owner. [antag_roles[1]] [!!length(U.owner_roles & antag_roles)]")
 		return !!length(U.owner_roles & antag_roles)
 	else
+		to_chat(world, "uplink owner, is in antag list? [antag_roles[1]] [player_is_antag_in_list(U.uplink_owner, antag_roles)]")
```

Comment:
Debug lines ?

Reply:
yes

Reply:
Okay, they are going to the world, everyone can hear it if something breaks.